frontend/javascripts/libs/input.ts

```diff
@@ -113,7 +157,14 @@ export class InputKeyboardNoLoop {
       },
       _.noop,
     ];
-    KeyboardJS.bind(...binding);
+    if (isExtendedCommand) {
+      KeyboardJS.withContext("extended", () => {
+        KeyboardJS.bind(...binding);
+      });
+    }
+    KeyboardJS.withContext("default", () => {
+      KeyboardJS.bind(...binding);
+    });
```

Comment:
In order to realize / implement the "extended keyboard mode" we use the [context feature of KeyboardJS](https://github.com/RobertWHurst/KeyboardJS#using-contexts). This might not be the way this feature was intended but it works :+1: